### PR TITLE
fix(developer): Add note about breaking UI change due to heading styles

### PR DESCRIPTION
**Pull request by susnux**
### ☑️ Resolves

* For: https://github.com/nextcloud/server/pull/43640

### 🖼️ Screenshots

image

### CODE REVIEW

File: developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_30.rst [hunk 1]
```diff
@@ -8,6 +8,13 @@ General
 Front-end changes
 -----------------
 
+Breaking changes
+^^^^^^^^^^^^^^^^
+
```
Comment (ChristophWurst) [inline developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_30.rst:13]: should this get its own heading?

Comment (susnux) [reply developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_30.rst:13]: Would you put it directly into the one above?

Comment (ChristophWurst) [reply developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_30.rst:13]: replace *Breaking changes* with *Font sizes* and the structure looks good imo

there might be other breaking changes in 30, that's why I would just not put the paragraph as *Breaking changes*

Comment (susnux) [reply developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_30.rst:13]: Done :white_check_mark:

**Review by ChristophWurst — APPROVED** [review]
:sunglasses:

**Pull request merged**